tools/build_pytorch_libs.py

```diff
@@ -181,7 +181,15 @@ def run_cmake(version,
         USE_REDIS=os.getenv('USE_REDIS'),
         USE_GLOG=os.getenv('USE_GLOG'),
         USE_GFLAGS=os.getenv('USE_GFLAGS'),
-        WERROR=os.getenv('WERROR'))
+        WERROR=os.getenv('WERROR'),
+        # openmp cmake flag
+        # If we setup MACOSX_DEPLOYMENT_TARGET in mac, we should also set the
+        # libomp library flags explicitly.
+        OpenMP_C_FLAGS=os.getenv('OpenMP_C_FLAGS'),
+        OpenMP_CXX_FLAGS=os.getenv('OpenMP_CXX_FLAGS'),
+        OpenMP_C_LIB_NAMES=os.getenv('OpenMP_C_LIB_NAMES'),
+        OpenMP_CXX_LIB_NAMES=os.getenv('OpenMP_CXX_LIB_NAMES'),
+        OpenMP_omp_LIBRARY=os.getenv('OpenMP_omp_LIBRARY'))
```

Comment:
This is not used anywhere. `OpenMP_libomp_LIBRARY` you used below is actually set by `FindOpenMP.cmake`.

Reply:
These changes are reverted.